### PR TITLE
SOLAS-5223: react-quiz - As a DEVELOPER I am able to style the quiz

### PR DESCRIPTION
**Pull request by 2creative**
https://bcdigital.atlassian.net/browse/SOLAS-5223

This story is to provide dynamic and static classes to some of the quiz components such as:
Quiz - Quiz container, Next Button
Question - Question Instruction, Question Answer List, 
Answer - Answer, Answer Button, 

Also the necessary tests were written for the above components

To Review:
npm install
npm Start
and go to localhost:3000 on your browser

**Note: Depends on #3. To see changes please look at [this diff](https://github.com/britishcouncil/react-quiz/compare/solas-5222-next-question...solas-5223-react-quiz-style-quiz?expand=1)**

To Run the tests:
npm test -- --watch
All should pass..

### CODE REVIEW

**Comment by LucjaWszola** [comment]
reviewing...

File: demo/quiz.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.rq-Answer-button:focus {
+  background-color: #333;
+}
+
+.rq-Answer-button--active {
+  background-color: #333;
+}
+
```
Comment (LucjaWszola) [inline demo/quiz.css:20]: Cannot we combine these two and not repeat the same bg color?

**Comment by LucjaWszola** [comment]
Looking good. 👍 

level

**Comment by LucjaWszola** [comment]
and... 19 tests passed! :)

**Comment by mnapthine** [comment]
Guys, I've had a quick look through and I'm happy to get these stories merged if you want to crack on and do that. 👍

File: demo/quiz.css (hunk 1 above)
Comment (andreadsw) [inline demo/quiz.css:20]: just a quick reminder that we don't use an uppercase on class names. It should be `.rq-answer`, `.rq-answer-button:focus` and `.rq-answer-button--active`.

**Comment by robinedman** [comment]
@LucjaWszola @mnapthine Thanks for reviewing! I'll merge it.

**Comment by andreadsw** [comment]
It works and looks good. 👍   19 tests are passed too.

firefox-quiz-component

**Pull request merged**